front-end/src/components/analysis-page/AnalysisPage.jsx

```diff
@@ -49,10 +49,29 @@ export class AnalysisPage extends Component {
             data-testid="analysis-page-main"
           >
             <div className="flex gap-y-10 w-full flex-col h-full">
-              <div className= "w-full bg-gray-100 pb-2">
+              <div className= "flow-root w-full bg-gray-100 pb-2">
+                {
+                    !this.state.showIntentMenu &&
+                    <nav>
+                      <button
+                        className="float-left m-auto px-5 py-2"
+                        onClick={this.openIntentMenu}
+                        aria-label="intent menu button"
+                        data-testid="intent-menu-button"
+                        aria-haspopup="menu"
+                        type="button"
+                      >
+                        <label
+                          className="cursor-pointer"
+                        >
+                          <Hamburger />
+                        </label>
+                      </button>
+                    </nav>
+                }
                 <div /* first line of className is tooltip styling */
                   className="relative before:z-10 before:absolute before:left-1/2 before:-bottom-3 before:w-max before:max-w-xs before:-translate-x-1/2 before:translate-y-full before:rounded-lg before:bg-gray-200 before:px-2 before:py-1.5 before:text-off-white before:invisible before:content-[attr(tooltip)] after:z-10 after:absolute after:left-1/2 after:-bottom-3 after:h-0 after:w-0 after:-translate-x-1/2 after:border-8 after:border-b-gray-200 after:border-l-transparent after:border-t-transparent after:border-r-transparent after:invisible hover:before:visible hover:after:visible
-                  absolute cursor-pointer m-3 sm:w-[11em] mx-auto"
+                  float-none cursor-pointer m-3 sm:w-[11em] mx-auto"
```

Comment:
```suggestion
                  float-none cursor-pointer m-3 w-[11em] mx-auto"
```
Removing the sm: will hopefully stop the issues with the title taking up too much space and the hamburger button being squished awkwardly in the top (it worked on my local so I'm hoping it will work on your commit as well)